src/opentracing/span.js

```diff
@@ -105,7 +105,7 @@ class DatadogSpan extends Span {
   _addTags (keyValuePairs) {
     try {
       Object.keys(keyValuePairs).forEach(key => {
-        this._spanContext._tags[key] = String(keyValuePairs[key])
```

Comment:
Hmm, is this going to have any other side effects? Or everything is going to be handled basically like this when it goes through `JSON.stringify()` anyways ?

Reply:
Everywhere it needs to be a string are already casting it to a string, so no change there. And when it goes through the formatter, it will be also get stringified in a safer way, so good there too.

Reply:
Cool, yeah, my concern was more about users manually setting tags as a non-string, so :+1: